### PR TITLE
fix: bugfix for warpcast provider

### PR DESCRIPTION
**Pull request by nuanyang233**
FW-250 FW-226 FW-268 FW-269 FW-270

https://mask.atlassian.net/browse/FW-250

### CODE REVIEW

File: src/components/Compose/ComposeContent.tsx [hunk 1]
```diff
@@ -23,7 +23,7 @@ export default function ComposeContent(props: ComposeContentProps) {
                         <div className=" mb-3 text-left text-[15px] text-fourMain">
                             <Trans>Replying to</Trans>
                             <span className="text-link"> @{post.author.handle} </span>
-                            <Trans>on Lens</Trans>
+                            <Trans>on {post.source}</Trans>
```
Comment (guanbinrui) [inline src/components/Compose/ComposeContent.tsx:26]:
```suggestion
                            <Trans>on {resolveSourceName(post.source)}</Trans>
```

File: src/hooks/useToggleFollow.tsx [hunk 2]
```diff
@@ -62,7 +62,7 @@ export function useToggleFollow(profile: Profile) {
                 <Select
                     value={followStateRef.current ? 'unfollow' : 'follow'}
                     _follow={`Followed @${handle} on ${source}`}
-                    _unfollow={`UnFollowed @${handle} on ${source}`}
+                    _unfollow={`Unfollowed @${handle} on ${source}`}
                     other={`Followed @${handle} on ${source}`}
```
Comment (guanbinrui) [inline src/hooks/useToggleFollow.tsx:66]: Enable locales.

Comment (nuanyang233) [reply src/hooks/useToggleFollow.tsx:66]: `<Select />` is a component of `lingui/marco`, i18n is enabled.

File: src/components/Compose/ComposeContent.tsx [hunk 3]
```diff
@@ -23,7 +24,7 @@ export default function ComposeContent(props: ComposeContentProps) {
                         <div className=" mb-3 text-left text-[15px] text-fourMain">
                             <Trans>Replying to</Trans>
                             <span className="text-link"> @{post.author.handle} </span>
-                            <Trans>on Lens</Trans>
+                            <Trans>on {resolveSourceName(post.source)}</Trans>
```
Comment (guanbinrui) [inline src/components/Compose/ComposeContent.tsx:27]: Merge as single `<Trans />`.

**Review by guanbinrui — APPROVED** [review]

**Pull request merged**